### ISSUE TITLE
sequana_coverage error: could not find or load the Qt platform plugin "xcb"

### ISSUE THREAD

**Issue by jdparker101** [sequana/sequana#481]
I am trying to use your recently released sequana_covereage. I installed sequana via conda like so:

```
$ conda create -n sequana python==3.5
$ source activate sequana
$ conda install sequana
```
Note that my conda channels are configured thus:

```
$ conda config --get channels
--add channels 'r'    # lowest priority
--add channels 'defaults'
--add channels 'conda-forge'
--add channels 'bioconda'   # highest priority
```

When I try to run sequana_coverage I get the following error message:
```
This application failed to start because it could not find or load the Qt platform plugin "xcb"
in "".

Available platform plugins are: minimal, offscreen, xcb.

Reinstalling the application may fix this problem.
Aborted (core dumped)
```

I've tried running with both
```
$ sequana_coverage --input test.bed -o
```

and

```
$ sequana_coverage --input test.bed -o--no-report
```

with the same results. I've also tried installing though `pip` and `python setup.py install`, but both installations failed.

I'm running on Xubuntu 16.

Any help would be appreciated.

**Comment by cokelaer** [comment]
Hi, could you make sure those lib are installed on your system:

```
apt-get install -y libsm-dev
apt-get install -y libxcomposite-dev
```

and let us know whether it works or get a different error message. thanks

**Comment by cokelaer** [comment]
@jdparker101 did the apt-get install solve your problem ?

**Comment by cokelaer** [comment]
@jdparker101 if you are under Linux, you can also use the singularity container (see http://sequana.readthedocs.io/en/master/installation.html#singularity) . Once you have the singularity container, you can simply use:
```
singularity exec sequana.img sequana_coverage 
```

**Comment by cokelaer** [comment]
I have released a new version 0.6.3.post1 on bioconda. 
Otherwise, a singularity container is also available as explained in the doc: http://sequana.readthedocs.io/en/master/installation.html#singularity
